build.gradle

```diff
@@ -43,4 +43,6 @@ dependencies {
 
 	//The main cotton api. We need a maven repo...
 	modCompile "com.github.CottonMC:cotton:master-SNAPSHOT"
+	
+	modCompile "com.google.code.findbugs:jsr305:3.0.2"
```

Comment:
Does this need to stick around? I don't see any uses of it in the code at the moment.

Reply:
any "@Nonnull" requires JSR. There is a nonnull in the spec now.
Benefits:
* Better Kotlin integration
* "CLASS" retention type requires the JVM to not load it at runtime, so it's literally zero-weight.